core/includes/image.inc

```diff
@@ -162,6 +174,18 @@ function image_get_info($filepath, $toolkit = FALSE) {
     if (isset($details) && is_array($details)) {
       $details['file_size'] = filesize($filepath);
     }
+    elseif (image_is_svg($filepath)) {
+      // Dimensions could be FALSE if the SVG is invalid, although we should
+      // have caught that on upload.
+      $dimensions = image_get_svg_dimensions($filepath);
+      $details = array(
+        'width'     => isset($dimensions['width']) ? $dimensions['width'] : IMAGE_SVG_DEFAULT_WIDTH,
```

Comment:
I guess, this extra check (isset) is unnecessary now that a FALSE returned by simplexml_load_string() is already caught in function image_get_svg_dimensions(), which could already make sure, valid dimensions are returned.

Reply:
I think this is resolved.